src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -125,19 +129,23 @@ public void On(string methodName, Type[] parameterTypes, Func<object[], Task> ha
             _handlers.AddOrUpdate(methodName, invocationHandler, (_, __) => invocationHandler);
         }
 
-        public async Task<ReadableChannel<object>> StreamAsync(string methodName, Type returnType, object[] args, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<ReadableChannel<object>> StreamAsync(string methodName, Type returnType, object[] args, CancellationToken cancellationToken = default)
         {
             return await StreamAsyncCore(methodName, returnType, args, cancellationToken).ForceAsync();
         }
 
         private async Task<ReadableChannel<object>> StreamAsyncCore(string methodName, Type returnType, object[] args, CancellationToken cancellationToken)
         {
+            if (cancellationToken.IsCancellationRequested)
```

Comment:
Not sure if this is needed and if it is just use `cancellationToken.ThrowIfCancellationRequested()`